src/box/index.h

```diff
@@ -252,6 +252,15 @@ box_tuple_extract_key(box_tuple_t *tuple, uint32_t space_id,
 
 /** \endcond public */
 
+/**
+ * Get packed position of tuple in index to pass it to box_select
```

Comment:
Please mention that the position is allocated on the fiber region.

Reply:
Done